actions/image_partition_action.go

```diff
@@ -102,6 +102,9 @@ checks in boot time. By default is set to `true` allowing checks on boot.
 ext2, ext3, ext4 and xfs.
 
 - partuuid -- GPT partition UUID string.
+can generate version 5 UUID using custom template function {{ uuid5 $namepsace $data }}
```

Comment:
```suggestion
can generate version 5 UUID using custom template function {{ uuid5 $namespace $data }}
```

Reply:
oops, thank for pointing out, I will correct the typo and update this PR.